components/Container/LayoutContainer.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import React from 'react';
+
+
+/**
+ * This is the base layout component. Our custom layout components should
+ * inherrit from this one. So that we have some common functionality for all
+ * layout containers.
+ */
+interface LayoutContainerProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
L11: @andrewebdev I think you need to export this interface, or you can move it to the shared `app/types` file because it is used in both LayoutWithPanel and ToolbarContainer.